presto-docs/src/main/sphinx/develop/worker-protocol.rst

```diff
@@ -112,6 +112,17 @@ output buffer zero.
 .. image:: worker-protocol-results.png
   :width: 600
 
+Presto Native Sidecar Endpoints
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following HTTP methods are used by the coordinator to fetch data from the
+Presto Native Sidecar process.
+
+* A ``GET`` on ``/v1/info/workerFunctionSignatures`` returns a json containing
+  the function metadata for all functions registered in the sidecar. The json
+  has the function names as keys, and a json array of function metadata, each
+  conforming to the ``protocol::JsonBasedUdfFunctionMetadata`` format.
```

Comment:
When we document this in OpenAPI, I think we should go into greater depth and actually describe the form of the JSON object.